pkg/svcat/service-catalog/broker.go

```diff
@@ -163,22 +173,43 @@ func (sdk *SDK) Register(brokerName string, url string, opts *RegisterOptions) (
 }
 
 // Sync or relist a broker to refresh its catalog metadata.
-func (sdk *SDK) Sync(name string, retries int) error {
+func (sdk *SDK) Sync(name string, opts ScopeOptions, retries int) error {
```

Comment:
same here, I think calling this something like 'scopeOpts' would be good.

Reply:
I can do that, but all the code in same file for other functions use 'opts'